definitions/npm/linked-list_v3.0.x/flow_v0.104.x-/linked-list_v3.0.x.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Type def from from source code of linked-list.
+ * this: https://github.com/wooorm/linked-list
+**/
+declare module 'linked-list' {
+  declare class Item {
```

Comment:
I'm surprised, you don't need to write `declare export class Item {` to be able to import?

Reply:
oh...
let me check it again

Reply:
I'm just super curious because I thought you do. But your tests don't seem to die 😅